raiden-dapp/src/components/AppHeader.vue

```diff
@@ -20,31 +24,50 @@
               width="36"
               contain
               aspect-ratio="1"
-              class="blockie"
+              class="app-header__top__content__blockie"
               :src="$blockie(defaultAccount)"
             ></v-img>
           </div>
         </div>
       </v-flex>
     </v-layout>
-    <v-layout class="row-2" align-center>
-      <v-flex lg6 md6 xs6>
-        <div class="address text-xs-left">
+    <v-layout class="app-header__bottom" align-center>
+      <v-flex xs6>
+        <div class="app-header__bottom__address text-xs-left">
           <v-tooltip bottom>
-            <template v-slot:activator="{ on }">
-              <span v-on="on"> {{ defaultAccount | truncate }}</span>
+            <template #activator="{ on }">
+              <span v-on="on">
+                {{ defaultAccount | truncate(8) }}
```

Comment:
Would it be useful to truncate only if width is smaller than a threshold? I don't think the address is worth it to be truncated if there's enough space

Reply:
It would expand way too much compared to the ETH amount on the other size though. 
now it looks close to the size of the eth amount on the other size.

Reply:
Also, the max size of the card is around 620px so the full address and the copy button will cross the middle of the card and get close to the amount.
![image](https://user-images.githubusercontent.com/2269732/61713912-ebb96080-ad59-11e9-8ac5-a237018f8720.png)